packages/flame_texturepacker/pubspec.yaml

```diff
@@ -12,8 +12,8 @@ topics:
   - spritesheet
 
 environment:
-  sdk: ">=3.6.0 <4.0.0"
-  flutter: ">=3.27.1"
+  sdk: ">=3.5.3 <4.0.0"
+  flutter: ">=3.24.3"
```

Comment:
This should match whatever we have in the root pubspec, since it will be overwritten by `melos bootstrap`.

Reply:
That's why the benchmark action fails, curious that no other action fails due to that...